pkg/cloud/services/compute/instance.go

```diff
@@ -142,50 +137,36 @@ func (s *Service) InstanceCreate(openStackCluster *infrav1.OpenStackCluster, mac
 	}
 	input.Networks = &nets
 
-	out, err := createInstance(s, clusterName, input)
+	out, err := createInstance(s, openStackMachine, clusterName, input)
 	if err != nil {
-		record.Warnf(openStackMachine, "FailedCreateServer", "Failed to create server %s: %v", input.Name, err)
 		return nil, err
 	}
-	record.Eventf(openStackMachine, "SuccessfulCreateServer", "Created server %s with id %s", out.Name, out.ID)
 	return out, nil
 }
 
-func createInstance(is *Service, clusterName string, i *infrav1.Instance) (*infrav1.Instance, error) {
-	// Get image ID
-	imageID, err := getImageID(is, i.Image)
-	if err != nil {
-		return nil, fmt.Errorf("create new server err: %v", err)
-	}
-
+func createInstance(is *Service, object runtime.Object, clusterName string, i *infrav1.Instance) (*infrav1.Instance, error) {
```

Comment:
Hm object is too generic imho as a variable name. Can we call it at least `eventObject` (in case that's the only context in which it's used)

Reply:
@hidekazuna Whad to you think about this one?

Reply:
@sbueringer Sorry I missed, updated.